docs/source/examples/petstore/petstore.all.rego.compiled

```diff
@@ -44,6 +44,13 @@ allow {
     re_match(`petstore.pet`, input.type)
 (input.status == "available")
 }
+allow {
+    seal_list_contains(input.subject.groups, `breeders_maltese`)
+    input.verb == `buy`
+    re_match(`petstore.pet`, input.type)
+(input.status == "reserved")
```

Comment:
we need to fix this eventually or run output through a reformatter.

Reply:
will do - created ticket https://github.com/infobloxopen/seal/issues/70